docs/BuildFromSource.md

```diff
@@ -139,6 +139,11 @@ source activate.sh
 code .
 ```
 
+Note that if you are using the "Remote-WSL" extension in VSCode, the environment is not supplied
+to the process in WSL.  You can workaround this by explicitly setting the environment variables
+in `~/.vscode-server/server-env-setup`.
```

Comment:
Which environment variables are you referring to? Also, is Remote-WSL the same as WSL2 here?

Reply:
Env vars are: `DOTNET_ROOT`, `DOTNET_MULTILEVEL_LOOKUP` and `PATH`.

"Remote-WSL" is the name of the VSCode extension.  It works with both WSL1 and WSL2.

Reply:
Are these environment variables listed elsewhere and what values they should be set to? I don't see them else in this document.